src/PiTop.MakerArchitecture.Expansion/MathHelpers.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Linq;
```

Comment:
These functions should move into the Algorithms section

Reply:
moved and moved tests too

Reply:
done